actions/class.Items.php

```diff
@@ -172,11 +172,20 @@ public function editItem()
                 $this->setData('id', $item->getUri());
             }
 
-            $formContainer = new SignedFormInstance($itemClass, $item, [FormContainer::CSRF_PROTECTION_OPTION => true]);
+            $itemUri = $item->getUri();
+            $hasWriteAccess = $this->hasWriteAccess($itemUri) && $this->hasWriteAccessToAction(__FUNCTION__);
+
+            $formContainer = new SignedFormInstance(
+                $itemClass,
+                $item,
+                [
+                    FormContainer::CSRF_PROTECTION_OPTION => true,
+                    FormContainer::IS_DISABLED => !$hasWriteAccess,
+                ]
+            );
             $myForm = $formContainer->getForm();
```

Comment:
If I am not mistaken, here you already persisted the data, not in the `isSubmited`, so basically it is saving without permissions.

Reply:
Checked and it is not happening. Thanks @shpran